### ISSUE TITLE
Allow arbitrary whitespace with comment-space-inside

### ISSUE THREAD

**Issue by MadLittleMods** [stylelint/stylelint#403]
[`comment-space-inside`](https://github.com/stylelint/stylelint/blob/master/src/rules/comment-space-inside/README.md) should only apply when content is on the same line as the start and end indicators(`/*` and `*/`) single line comments

Adding a `always-single-line` like other rules already have isn't quite enough to cover every basis, but would satisfy my need.

Currently this throws a warning/error if `comment-space-inside: 'always'`:
```css
.foo {
	/*
	opacity: 0;
	pointer-events: none;
	*/
}
```

If content is on same line as `/*` or `*/` then the space should be necessary but only to appropriate piece. I expect this to pass when `comment-space-inside: 'always'`:
```css
.foo {
	/* opacity: 0;
	pointer-events: none;
	*/
}
```

**Comment by jeddy3** [comment]
Sounds sensible to me. Thanks for reporting!

**Comment by davidtheclark** [comment]
Question here: What constitutes a multi-line comment? Just the text, or also the space between `/*` and `*/` and text?

Is the following single-line?

```css
/*
this is only one line of text
*/
```

Or does it have to be this?

```css
/* this is only one line of text */
```

**Comment by davidtheclark** [comment]
If only the text determines single/multi line, then we need to have both `newline` and `space` rules have all the options (always, never, always-single-line, always-multi-line, etc.). Right?

**Comment by MadLittleMods** [comment]
@davidtheclark I would consider multiline to be all space between `/*` and `*/` and text.

```
/*
this is should be considered a multiline comment
*/

/*
this is should be considered a multiline comment */

/* this is should be considered a multiline comment
*/
```

**Comment by davidtheclark** [comment]
@jeddy3 : Are you working on this one? I don't want to step on your toes if you are (saw that you assigned the wip label), but am happy to take care of it if you're not.

**Comment by jeddy3** [comment]
>  Are you working on this one?

@davidtheclark Nope. The wip label is for @MadLittleMods' PR over in https://github.com/stylelint/stylelint/pull/406. So feel free to pick this one up. 

I'm now wondering if @MadLittleMods was right with his choice of solution in his initial PR... I suggested breaking it out into two rules `comment-space-inside` and `comment-newline-side`, but now I'm wondering if we don't need that level of detail for comments. Especially as we're introducing the concept of `*-whitespace-*` with the upcoming `function-whitespace-after` in https://github.com/stylelint/stylelint/issues/451. What are your thoughts?

If we go down this route than I suggest, as we've done with `function-space-after`, keep this rule named `comment-space-inside` but support newlines also. And then rename the rule to `comment-whitespace-inside` for `3.0.0`.

**Comment by davidtheclark** [comment]
I'm good with this. I'll put together a quick PR.
